package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@salsify/ember-power-select",
-  "version": "2.0.8-beta.0-0.3.2",
+  "version": "2.2.1-0.3.2",
```

Comment:
Can we drop the `-0.3.2` as [ember-power-select does](https://github.com/cibernox/ember-power-select/blob/master/package.json#L3)?

Reply:
this is our fork version. It indicates the upstream version as well as the nature of our changes on top of upstream. [here's an explanation](https://github.com/semver/semver/issues/17#issuecomment-49895206) of a similar versioning scheme

Reply:
Actually [here's the explanation](https://salsify.atlassian.net/wiki/spaces/ENG/pages/96147257/Semantic+Versioning+for+Forks) for our versioning scheme for this fork.